3p/iframe-transport-client.js

```diff
@@ -83,6 +83,19 @@ export class IframeTransportClient {
   }
 
   /**
+   * Sends a message back to the creative
+   * @param {string} vendor The name of the 3p vendor used in the
+   * <amp-analytics> tag
+   * @param {string} creativeId An ID uniquely identifying which creative
+   * shall receive the event
+   * @param {!Object<string,string>} response
+   */
+  sendMessageToCreative(vendor, creativeId, response) {
+    this.client_./*OK*/sendMessage(MessageType.IFRAME_TRANSPORT_RESPONSE,
```

Comment:
What's the policy on code review for sensitive APIs?

Reply:
I'm not sure exactly the general policy is, but in this case it was discussed in design meetings/docs/emails.